lib/msprime.c

```diff
@@ -4462,6 +4469,13 @@ msp_run_dtwf(msp_t *self, double max_time, unsigned long max_events)
             }
             assert(mig_tmp[j] == 0);
 
+            // Must check that migration matrix is right stochastic in the main
```

Comment:
perhaps say "must have rows that sum to <= 1" instead of "right stochastic", to be less jargoney